src/Components/ActivationKeys/index.js

```diff
@@ -1,31 +1,54 @@
 import React from 'react';
```

Comment:
Could you add tests for this file?  Without having looked too carefully, I'd assume the tests would be similar to the ones [here](https://github.com/RedHatInsights/subscription-inventory-ui/blob/main/src/pages/SubscriptionInventoryPage/__tests__/SubscriptionInventoryPage.test.tsx).

Reply:
Seeing the tests at ActivationKeys.test.js made me think the ActivationKeys component should probably be moved out to its own file to match everything else.  Then this index.js file can just be used for exports like our other index.js files.